src/workers/build/mod.rs

```diff
@@ -0,0 +1,151 @@
+mod krate;
+
+use std::fs;
+use std::io::BufReader;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+use super::{Resource, Script, WasmModule, Worker};
```

Comment:
I think WorkerBundle could become `/publish/mod.rs` that has `impl` to return the multipart form from a `Worker`, and get rid of the separate data structure

Reply:
I disagree, separating the domain entity of a Worker from the implementation of our API requests provides value.

Reply:
I do think it would make sense to further improve the organization of the code, though.